scripts/explicit_kwargs_checker_test.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For details on how to write such tests, please refer to
+# https://github.com/oppia/oppia/wiki/Writing-Tests
```

Comment:
The title of this wiki page is too generic. It looks like a page for writing tests, rather than a page for writing tests for custom lint checkers. Might want to rename it to make it more specific.

Reply:
Shall I change it to `Writing-Tests-For-Pylint`?

Reply:
Yes, that's a good name.